tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -348,3 +346,195 @@ def test_del_obj_tag_40436(self):
+                                   file_path=self.file_path1,
+                                   object_name=self.object_name,
+                                   versions_dict=self.versions)
+        latest_v = self.versions[self.object_name]["version_history"][-1]
+        LOGGER.info("Step 2: Successfully uploaded object %s on  "
+                    "bucket %s with version ID %s", self.object_name, self.bucket_name, latest_v)
+
+        LOGGER.info("Step 3: Perform PUT Object Tagging for %s with a tag key-value pair",
+                    self.object_name)
+        resp = s3_ver_tlib.put_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name,
+                                              version_tag=self.ver_tag,
+                                              versions_dict=self.versions)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 3: Performed PUT Object Tagging for %s with a tag key-value pair",
+                    self.object_name)
+
+        LOGGER.info("Step 4: Perform GET Object Tagging for %s with versionId=%s",
+                    self.object_name, latest_v)
+        put_tag = self.ver_tag[self.object_name][latest_v][-1]
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name,
+                                              version_id=latest_v)
+        assert_utils.assert_true(resp[0], resp)
+        get_tag = resp[1][0]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+        LOGGER.info("Step 4: Performed GET Object Tagging for %s with versionId=%s is %s",
+                    self.object_name, latest_v, get_tag)
+
+        LOGGER.info("Step 5: Perform DELETE Object %s to creates delete marker with versionId=%s",
+                    self.object_name, latest_v)
+        s3_ver_tlib.delete_version(s3_ver_test_obj=self.s3_ver_obj,
+                                   bucket_name=self.bucket_name,
+                                   object_name=self.object_name,
+                                   versions_dict=self.versions,
+                                   check_deletemarker=True)
+        dm_id = self.versions[self.object_name]["delete_markers"][0]
+        assert_utils.assert_in("No Content", resp[1].message)
+        LOGGER.info("Step 5: Performed DELETE Object tagging %s "
+                    " delete_marker id %s, with versionId=%s",
```

Comment:
Or we can just say creates delete marker with versionId %s, dm_id.

This dm_id it self called versionID

Reply:
Ok, Statement updated